OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -285,7 +286,7 @@ private List<string> GetCollisions(ReadOnlyCollection<long> hashes)
                 {
                     combos.Add(_comboRegistry[hash].FullName);
                 }
-                if (hash < ModInputLibrary.MaxUsefulKey && _gameBindingCounter[hash] > 0)
+                else if (hash < ModInputLibrary.MaxUsefulKey && _gameBindingCounter[hash] > 0)
```

Comment:
Ah, was it wrong? :)

Reply:
Technically both can't be true at the same time, so either is technically right. However with 'else' second if won't be called, doesn't really significantly affect performance though